x-pack/legacy/plugins/siem/public/pages/hosts/hosts_navigations.tsx

```diff
@@ -25,9 +25,9 @@ import { ESTermQuery } from '../../../common/typed_json';
 import { HostsTableType } from '../../store/hosts/model';
 import { StatefulEventsViewer } from '../../components/events_viewer';
 
-const getTabsOnHostsUrl = (tabName: HostsTableType) => `#/hosts/${tabName}`;
+const getTabsOnHostsUrl = (tabName: HostsTableType) => `#/link-to/hosts/${tabName}`;
```

Comment:
The reason I remove `link-to` here was because it always does a redirect in `link-to component` and I don't want Kpis to be refresh while sub-tab changed.